sub-packages/bionemo-geometric/src/bionemo/geometric/atom_featurizers.py

```diff
@@ -0,0 +1,398 @@
+        Rb0 = self.pt.GetRb0(atomic_num)
+        Rb0 = Rb0 / 2.4  # Standard scaler
+
+        Rcovalent = self.pt.GetRcovalent(atomic_num)
+        Rcovalent = (Rcovalent - 0.28) / (2.6 - 0.28)  # Standard scaler
+
+        Rvdw = self.pt.GetRvdw(atomic_num)
+        Rvdw = (Rvdw - 1.2) / (3.0 - 1.2)
+
+        return [Rb0, Rcovalent, Rvdw]
+
+    def get_features(self, atom: Chem.Atom) -> list[float]:
+        """Returns features of the atom."""
+        return self.get_period(atom) + self.get_group(atom) + self.get_atomic_radii(atom)
+
+
+class ElectronicPropertyFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom by its electronic properties.
+
+    This class computes electronic properties like electronegativity, ionization energy, and electron affinity.
+    """
+
+    def __init__(self, data_file=None) -> None:
+        """Initializes PeriodicTableFeaturizer class.
+
+        Args:
+            data_file: Path to the data file.
+        """
+        if data_file is None:
+            # Use default
+            root_path = Path(__file__).resolve().parent
+            data_file = root_path / "data" / "electronic_data.csv"
+        self.data_df = pd.read_csv(data_file).set_index("AtomicNumber")
+
+        self.pauling_en_dict = self.data_df["Electronegativity"].to_dict()
+        self.ie_dict = self.data_df["IonizationEnergy"].to_dict()
+        self.ea_dict = self.data_df["ElectronAffinity"].to_dict()
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return 3
+
+    def get_pauling_electronegativity(self, atom: Chem.Atom) -> list[float]:
+        """Returns electronegativity on Pauling scale."""
+        atomic_num = atom.GetAtomicNum()
+        en = self.pauling_en_dict[atomic_num]
+        en = (en - 0.7) / (3.98 - 0.7)
+        return [en]
+
+    def get_ie(self, atom: Chem.Atom) -> list[float]:
+        """Returns ionization energy."""
+        atomic_num = atom.GetAtomicNum()
+        ie = self.ie_dict[atomic_num]
+        ie = (ie - 3.894) / (24.587 - 3.894)
+        return [ie]
+
+    def get_ea(self, atom: Chem.Atom) -> list[float]:
+        """Retuns electron affinity."""
+        atomic_num = atom.GetAtomicNum()
+        ea = self.ea_dict[atomic_num]
+        ea = (ea - 0.079) / (3.617 - 0.079)
+        return [ea]
+
+    def get_features(self, atom: Chem.Atom) -> list[float]:
+        """Returns features of the atom."""
+        return self.get_pauling_electronegativity(atom) + self.get_ie(atom) + self.get_ea(atom)
+
+
+class ScaffoldFeaturizer(BaseFeaturizer):
+    """Class for featurizing atom based on whether it is present in Bemis-Murcko scaffold."""
+
+    def __init__(self):
+        """Initializes ScaffoldFeaturizer class."""
+        pass
+
+    @property
+    def n_dim(self) -> int:
+        """Returns dimensionality of the computed features."""
+        return 1
+
+    def compute_features(self, mol):
```

Comment:
Seems like the user has to manually call `compute_features` before `get_features`. Not sure if there is any perf concern on set property on rdkit mol object. 
And the `compute_features` is for all atoms/bonds, while `get_features` is for a specific atom/bond, better to unify them, like 
```
def get_features(self, mol, atom_idx_list = None):
    prop = self.compute_features(mol)
    if atom_idx_list is None:
        # return features for all atoms
    else:
        # return features for selected atoms
    
```
 similar for bond type features. But I don't know if there are other cases not covered here. No need to refactor if you don't think this is reasonable.